### PR TITLE
Test base and button resource controller

### PR DESCRIPTION
**Pull request by idekker**

### CODE REVIEW

File: aiohue/v2/controllers/base.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 import asyncio
 from asyncio.coroutines import iscoroutinefunction
 from collections.abc import Callable, Iterator
+
+# pylint: disable=no-name-in-module
```
Comment (marcelveldt) [inline aiohue/v2/controllers/base.py:7]: why is this ?

Comment (idekker) [reply aiohue/v2/controllers/base.py]: Don't know. Probably some lint warning that came up with the original changes still included. I removed it and still passes.

**Comment by marcelveldt** [comment]
Thanks again for the help. Next time ping me on discord if the review takes too long. I do get about 300 notifications from Github per day so I quickly miss one ;-)

**Review by marcelveldt — APPROVED** [review]
LGTM, Thanks @idekker !

**Pull request merged**